src/test/scala/com/databricks/spark/csv/CsvSuite.scala

```diff
@@ -160,6 +160,8 @@ class CsvSuite extends FunSuite {
         |OPTIONS (path "$carsFile", header "true")
       """.stripMargin.replaceAll("\n", " "))
 
+    sql("select * from carsTable").collect().foreach { println(_) }
```

Comment:
Is this left from debugging? If those `println` are in fact needed, `.foreach(println)` could be used.